packages/compiler/test/server/completion.test.ts

```diff
@@ -2017,13 +2017,13 @@ describe("identifiers", () => {
        */
       id: int16;
       /**
-       * my log context
+       * my log ctx
        */
-      context: MyLogContext<string>[];
+      ctx: MyLogContext<string>[];
```

Comment:
filed an issue for this #6756, problem is autocomplete does always wrap keyword in quotes as its not context aware. This is not critical as the formatter will fix it after